pom.xml

```diff
@@ -711,7 +710,7 @@
     <!-- Start: dependency for Google PubSub Streaming Source -->
     <dependency>
       <groupId>org.apache.bahir</groupId>
-      <artifactId>spark-streaming-pubsub_2.11</artifactId>
+      <artifactId>spark-streaming-pubsub_2.12</artifactId>
       <version>2.4.0</version>
```

Comment:
Should this be 2.4.0?

Reply:
The only library using scala 2.12 is 2.4.0: https://mvnrepository.com/artifact/org.apache.bahir/spark-streaming-pubsub_2.12